healthcare/api-client/v1/dicom/dicom_stores_test.py

```diff
@@ -0,0 +1,221 @@
+# Copyright 2018 Google LLC All Rights Reserved.
```

Comment:
nit: Just to confirm, are these all files that are copied?

Reply:
Yes.